src/framework/vst/internal/vstconfiguration.h

```diff
@@ -23,17 +23,22 @@
 #ifndef MU_VST_VSTCONFIGURATION_H
 #define MU_VST_VSTCONFIGURATION_H
 
+#include "modularity/ioc.h"
+#include "iglobalconfiguration.h"
+
 #include "ivstconfiguration.h"
 
 namespace mu::vst {
 class VstConfiguration : public IVstConfiguration
 {
+    INJECT(vst, framework::IGlobalConfiguration, globalConfig)
 public:
     void init();
 
     io::paths_t userVstDirectories() const override;
     void setUserVstDirectories(const io::paths_t& paths) override;
     async::Channel<io::paths_t> userVstDirectoriesChanged() const override;
+    io::path_t knownPluginsList() const override;

```

Comment:
What do you think about knownPluginsFile?

Reply:
actually, forgot to rename it